test/common_methods_invocations.py

```diff
@@ -243,7 +243,7 @@ def method_tests():
         ('sinh', (), NO_ARGS, 'scalar', (True,)),
         ('cosh', (S, S, S), NO_ARGS, '', (True,)),
         ('cosh', (), NO_ARGS, 'scalar', (True,)),
-        ('abs', (S, S, S), NO_ARGS, '', (True,)),
+        ('abs', (L, L, L), NO_ARGS, '', (True,)),
```

Comment:
wait, are these being used for benchmarking?  These tests are really slow when run under test_autograd and they don't help correctness.

Reply:
I can change them back. I'm just trying to get them to be large enough to make sure they down the vector path

Reply:
Well we should definitely have some tests for the vector path

Reply:
We can just make it a slow test (and maybe only the forward?).

Reply:
My decision here is to just leave `common_method_invocations` as it is on master and rely on the `test_torch` tests to ensure the functionality of this PR